llvm/include/llvm/Analysis/IR2Vec.h

```diff
@@ -163,6 +163,14 @@ class Vocabulary {
   static constexpr unsigned MaxOperandKinds =
       static_cast<unsigned>(OperandKind::MaxOperandKind);
 
+public:
```

Comment:
What's the point of moving these declarations? Keep them in the same order but also make the `getVocabKeyFor*` functions part of the public interface?

Reply:
I just prefer to have constructor in the top :)